Felienne/hedy
# [FEATURE] Enables teachers to enable/disable the quiz

**Description**
In this PR we re-introduce the feature for teachers to enable/disable the quiz in a class. This was already possible when we still view the quiz from an adventure perspective. However, with recent clean-ups we dynamically load the quiz into the tabs; teachers can no longer enable/disable the quiz like an adventure. We fix this by adding the "hide quiz" setting to the other_settings tab within the customizations. We also slightly improve the UI so the buttons are nog longer at the bottom of the page in the case of a lot of open levels.

**Fixes**
This PR fixes #2558.

**How to test**
Make sure you are a teacher and have a class. Navigate to customizations and verify the new UI and the added option for "hiding the quiz". Verify that this is working as expected.
